wgpu-types/src/lib.rs

```diff
@@ -841,6 +841,8 @@ bitflags::bitflags! {
         const DEBUG = 1 << 0;
         /// Enable validation, if possible.
         const VALIDATION = 1 << 1;
+        /// Don't pass labels to wgpu-hal.
+        const DISCARD_HAL_LABELS = 1 << 2;
```

Comment:
I feel like this should be a positive flag that is on in the debugging configuration and debug mode (by default)

Reply:
My expectation is that the majority of people won't use this flag (if perf is what you care about you'll want to filter out label before passing them to wgpu-core). It's formulated this way so that `InstanceFlags::empty()` remains a good default and people opt into filtering labels by adding the bit.

Reply:
I see @cwfitzgerald's point - positive flags are generally less confusing that negative flags - but I also see Nical's point that all the extant flags are non-default features.

In the interest of progress, I'm going to mark this conversation as resolved. @cwfitzgerald, if you feel strongly about it, I'll write the PR to flip it.

Reply:
Don't feel particularly strongly, lets land as is.